src/app/project/comments/comments.component.ts

```diff
@@ -174,20 +193,27 @@ export class CommentsComponent implements OnInit, OnDestroy {
     this.getPaginatedComments(this.tableParams.currentPage);
   }
 
-  public goToItem(item) {
+  /**
+	 * Event handler for a user click on a document
+   * 
+	 * @param item The item that's been clicked on, either a document or external link entry
+	 * @returns {Observable<Object>}
```

Comment:
You can probably just cut this line altogether as the return type is documented is on line 202

Reply:
Return line cut